lib/valkyrie/persistence/fedora/persister.rb

```diff
@@ -99,5 +102,26 @@ def save_reference_to_resource(resource, alternate_resources)
 
         resource
       end
+
+      # @note Fedora's last modified response is not granular enough to produce an effective lock token
+      #   therefore, we use the same implementation as the memory adapter. This could fail to lock a
+      #   resource if Fedora updated this resource between the time it was saved and Valkyrie created
+      #   the token.
```

Comment:
I appreciate the need for finer-grained locking than Fedora's HTTP-date based 1-second granularity.  But maybe we should fall back on that, so we had at least that much protection against updates from non-Valkyrie clients?

Reply:
8/6/18: We decided that the current implementation is good enough to close this ticket, and created a separate ticket for using Fedora's `If-Unmodified-Since` header (#522)